doc/src/pair_coeff.rst

```diff
@@ -34,20 +34,28 @@ atom types.  The number and meaning of the coefficients depends on the
 pair style.  Pair coefficients can also be set in the data file read
 by the :doc:`read_data <read_data>` command or in a restart file.
 
-I and J can be specified in one of two ways.  Explicit numeric values
-can be used for each, as in the first example above.  I <= J is
-required.  LAMMPS sets the coefficients for the symmetric J,I
-interaction to the same values.
-
-A wildcard asterisk can be used in place of or in conjunction with the
-I,J arguments to set the coefficients for multiple pairs of atom
-types.  This takes the form "\*" or "\*n" or "n\*" or "m\*n".  If N = the
-number of atom types, then an asterisk with no numeric values means all
-types from 1 to N.  A leading asterisk means all types from 1 to n
-(inclusive).  A trailing asterisk means all types from n to N
-(inclusive).  A middle asterisk means all types from m to n
-(inclusive).  Note that only type pairs with I <= J are considered; if
-asterisks imply type pairs where J < I, they are ignored.
+I and J can be specified in one of several ways.  Explicit numeric
+values can be used for each, as in the first example above.  Or, one
+or both of the types in the I,J pair can be a type label, which is an
+alphanumeric string defined by the :doc:`labelmap <labelmap>` command
+or in a section of a data file read by the :doc:`read_data
+<read_data>` command, and which converts internally to a numeric type.
+In either case I <= J is required.  Interally, LAMMPS will set
```

Comment:
@sjplimp I removed I <= J requirement for pair_coeff, so that type labels can be used without knowing their numeric counterpart